### ISSUE TITLE
Push last version to Grafana Labs

### ISSUE THREAD

**Issue by felipetomm** [michaeldmoore/michaeldmoore-multistat-panel#63]
Hello @michaeldmoore!
Currently in Grafana Labs the version is 1.4.1. Do you have a prevision to push the last version (1.7.0) to they?

**Comment by michaeldmoore** [comment]
It seems to be stuck in the review cycle - I think they've overlooked it
for some (no?) reason.  Let me chase them up.  In the meantime, you should
be able to pull the release directly from my github repo.

**Comment by felipetomm** [comment]
Yep, i can pull from github, but this is go in against of my infra.. In my test server i will do this!
For production, I will wait before proceed. Thank's for your attention and fast answer!

**Comment by michaeldmoore** [comment]
Agreed.  I sent a wake up email to Marcus O.  Hopefully, we'll see some
action in the next few days.
